frida-gum/src/lib.rs

```diff
@@ -79,3 +83,11 @@ impl Drop for Gum {
 }
 
 pub struct NativePointer(pub *mut c_void);
+
+impl fmt::Debug for NativePointer {
```

Comment:
We should instead have a feature called `derive-debug` or likewise and use that for `#[derive(Debug)]` on our structs. We can deal with that later, for now just uncheck this code.

Reply:
Ok.